src/db/schema.ts

```diff
@@ -14,11 +14,11 @@ import {
 export const usersTable = pgTable(
   "users",
   {
-    userId: uuid("userId").defaultRandom().notNull().unique(),
-    ntuEmail: varchar("ntuEmail", { length: 63 }).notNull().unique(),
+    userId: uuid("userid").defaultRandom().notNull().unique(),
```

Comment:
What problems will we encounter if we use `userId` in the database?

Reply:
Please look at the commit messages.